packages/truffle-compile/profiler.js

```diff
@@ -189,7 +189,7 @@ module.exports = {
       var compilationTargets = [];
 
       // Load compiler
-      var supplier = new CompilerSupplier(options.compiler)
+      var supplier = new CompilerSupplier(options.compilers)
       supplier.load().then(solc => {
```

Comment:
Should this be `options.compilers.solc`, like in `truffle-compile/index.js`?

Reply:
Oh my god, good catch.